frontend/package.json

```diff
@@ -63,12 +62,10 @@
     "typescript": "^3.8.3"
   },
   "scripts": {
-    "start": "npm run buildTypeCheckers && run-p start-web start-server",
-    "start-web": "react-scripts start",
-    "start-server": "nodemon ./dev_proxy/index",
+    "start": "npm run buildTypeCheckers && react-scripts start",
     "build": "react-scripts build",
     "buildTypeCheckersClean": "rm -rf \"src/lib/services/generated\" && mkdir \"src/lib/services/generated\"",
-    "buildTypeCheckersGenerate": "ls src/lib/services/*.ts | xargs ts-interface-builder -o \"src/lib/services/generated\" --suffix Types",
+    "buildTypeCheckersGenerate": "ls src/lib/services/Fetch*.ts | xargs ts-interface-builder -o \"src/lib/services/generated\" --suffix Types",
```

Comment:
Because my services that had nothing related to the rest stuff were having generated files being created. Only the rest services need generated the generated stuff right?

Reply:
Yeah I noticed that too, but I think the solution to that is to put them in different folders. Was going to re-evaluate the folder structure once we had a few more services in there, then we can do /rest/*.ts or something like that.